modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -404,6 +404,28 @@ public ObjectDefinition putObjectDefinition(
 					serviceBuilderObjectField.getBusinessType(),
 					ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)) {
 
+				_objectFieldLocalService.updateObjectField(
+					serviceBuilderObjectField.getExternalReferenceCode(),
+					GetterUtil.getLong(
+						serviceBuilderObjectField.getObjectFieldId()),
+					contextUser.getUserId(),
+					serviceBuilderObjectField.getListTypeDefinitionId(),
+					objectDefinitionId,
+					serviceBuilderObjectField.getBusinessType(), null, null,
+					serviceBuilderObjectField.getDBType(),
+					serviceBuilderObjectField.getDefaultValue(),
+					serviceBuilderObjectField.getIndexed(),
+					serviceBuilderObjectField.getIndexedAsKeyword(),
+					serviceBuilderObjectField.getIndexedLanguageId(),
+					serviceBuilderObjectField.getLabelMap(),
+					serviceBuilderObjectField.getName(),
+					serviceBuilderObjectField.getRequired(),
+					GetterUtil.getBoolean(serviceBuilderObjectField.getState()),
+					serviceBuilderObjectField.getSystem(),
+					_objectFieldSettingLocalService.
+						getObjectFieldObjectFieldSettings(
+							serviceBuilderObjectField.getObjectFieldId()));
+
```

Comment:
@seltong How about creating the `updateDBTableName` method in `localService`? Since here we only need to update the `dbTableName` ...